setup.py

```diff
@@ -51,7 +51,7 @@
         package_name: ['docker/*.*', 'mixins/*.*'],
     },
     install_requires=[
-        'docker>=2,<3',
+        'docker>=2,<5',
```

Comment:
I've been thinking about upgrading this, the old version requirement is from an old situation that no longer applies. Perhaps we could just make this

```suggestion
        'docker==4.*',
```